Dockerfile.debug

```diff
@@ -0,0 +1,42 @@
+FROM golang:1.15.2 AS build
```

Comment:
Since we now have multi-arch support for our container image, it'd be nice to support that here as well - see recent commit https://github.com/projectcontour/contour/commit/baa89ee5b856b4ba9603caf7fa346afc6eef5b7f#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557